File: snippets/py/array/remove clear/remove.py
array = [5, 4, 3, 2, 1]
index = 1  # 0: first item, 1: second item...
print(array)  # [5, 4, 3, 2, 1]
del array[index]  # ≡
print(array)  # [5, 3, 2, 1]

# many items:
array = [5, 4, 3, 2, 1]
start = 1  # 0: first item, 1: second item...
count = 3
print(array)         # [ 5, 4, 3, 2, 1 ]
removedItems = array[start:start + count]  # ≡
del array[start:start + count]             # ≡
print(array)         # [ 5, 1 ]
print(removedItems)  # [ 4, 3, 2 ]

File: snippets/js/array/remove clear/remove.js
// many items:
var array = [5, 4, 3, 2, 1];
var start = 1;  // 0: first item, 1: second item...
var count = 3;
console.log(array);  // [ 5, 4, 3, 2, 1 ]
array.splice(start, count);  // ≡
console.log(array);  // [ 5, 1 ]